src/main/java/io/jenkins/plugins/opentelemetry/job/cause/GitHubPushCauseHandler.java

```diff
@@ -12,17 +12,21 @@
 import javax.annotation.Nonnull;
 
 @Extension(optional = true, dynamicLoadable = YesNoMaybe.YES)
-public class GitHubPushCauseHandler extends AbstractCauseHandler {
+public class GitHubPushCauseHandler implements CauseHandler {
 
     @Override
     public boolean isSupported(@Nonnull Cause cause) {
         return isGitHubPushCause(cause);
     }
 
+    protected boolean isGitHubPushCause(Cause cause) {
+        return cause instanceof com.cloudbees.jenkins.GitHubPushCause;
```

Comment:
```suggestion
        return cause.getClass().getName().equals("com.cloudbees.jenkins.GitHubPushCause");
```

Reply:
Probably this can be merged into isSupported

Reply:
Fixed slightly differently with https://github.com/jenkinsci/opentelemetry-plugin/pull/208/commits/f3a999c7f9a2a1929185d27c7bf2f7f44486b388